specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/IntegrationRuntime.json

```diff
@@ -146,6 +146,22 @@
             "name": "IntegrationRuntimeLicenseType",
             "modelAsString": true
           }
+        },
+        "customSetupScriptProperties": {

```

Comment:
nit: For consistency, you can name the model with PascalCase and name the properties with camelCased letters.

Reply:
This is consistent, properties within definition are start with low case, so using low case here.
And definition are start with upper case, refer to link 208